### ISSUE TITLE
Custom attribute value created in integration tests fixtures product not appears in elasticsearch

### ISSUE THREAD

**Issue by magento-engcom-team** [magento/magento2#30261]
* There is a product created with custom attribute value in integration tests fixture:Magento/Catalog/*files/products_with_layered_navigation*attribute.php
* When this product created and product data appeared in elasticsearch - there is no custom attribute (test_configurable) data in ElasticSearch.

This attribute data appears in Elastic Search Only after execution of indexer:reindex in console 

Note from comments:
{quote}
Custom attribute data (default value) doesn't save only for the first product in the fixture. This data doesn't appear in Elastic Search after manual reindexing. The possible cause of this issue may be calling \Magento\Eav\Model\Entity\AbstractEntity::walkAttributes method before loading attribute data.
{quote}

**Comment by magento-engcom-team** [comment]
The issue was exported from the internal JIRA. The link to the original JIRA issue: https://jira.corp.magento.com/browse/MC-36768

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @magento-engcom-team 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-36768` were created

**Issue Available**: @magento-engcom-team, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @sdzhepa 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-36768` were created

**Issue Available**: @sdzhepa, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by sdzhepa** [comment]
The issue was closed by mistake.
at this moment Magento team has fixed it, internal PR is ready(CR and QA passed) but not merged into the main code-base.
As soon as it will be merged related commits can be found using search based on internal ticket key MC-36768
Example: https://github.com/magento/magento2/search?q=MC-36768&type=commits

Sorry for inconvenience

**Comment by zakdma** [comment]
Internal ticket MC-36768 merged into mainline

**Comment by sdzhepa** [comment]
The related internal Jira ticket `MC-36768` was closed as `Fixed`.
Related commits can be found by search https://github.com/magento/magento2/search?q=MC-36768&type=commits
